### PR TITLE
comment

### PR DESCRIPTION
**Pull request by veniapputrii**
testing

### CODE REVIEW

File: 1. Decision Tree (beginners).ipynb [hunk 1]
```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```
Comment (veniapputrii) [inline 1. Decision Tree (beginners).ipynb:8]: haha

**Review by veniapputrii — COMMENTED** [review]
testing 2

File: 1. Decision Tree (beginners).ipynb [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample = {\n",
+    "    'Weather' : ['Sunny', 'Sunny', 'Overcast', 'Rain', 'Rain', 'Overcast', 'Sunny', 'Sunny', 'Rain', 'Sunny', 'Overcast', 'Overcast', 'Rain'],\n",
```
Comment (BalazsJava) [inline 1. Decision Tree (beginners).ipynb:30]: What a nice code, keep it up!

Comment (veniapputrii) [reply 1. Decision Tree (beginners).ipynb:30]: haha

**Review by BalazsJava — CHANGES_REQUESTED** [review]
Testing change request.

**Pull request merged**